### DISCUSSION TITLE
Filtering by Time Trace value

### DISCUSSION THREAD

**Discussion by TransitionMetal** [PicoQuant/snAPI#17] [Q&A]
Hello,
Is it possible to filter the data from .tpu file if the TimeTrace value is out of limit? Like in the QuCoa where one can limit the record by start and stop time and by min and max value for TimeTrace. Thank you.

**Comment by tpoint75** [comment]
No, not at the moment. But maybe we can make a manipulator, that filters the data, if the countrate exceeds a limit.
Can you explain what exactly you want to do?

**Reply by TransitionMetal** [reply]
Thank you. I'm measuring the antibunching of photons from single quantum dot and due to drifts it can move out of detection spot causing drop of the signal. I would like to filter out the data where the signal is too low and have a possibility to cut the record from the beginning and the end if the signal is unstable there. Like in QuCoa.
